analysis/cQED_dataAnalysis/tomo_gate_set.m

```diff
@@ -9,14 +9,36 @@
 Z = [1 0; 0 -1];
 I = eye(2);
 
+p = inputParser;
+addParameter(p, 'type', 'Clifford', @ischar) %prepared states/meas. axes
+addParameter(p, 'prep_meas', 1, @isnumeric) %1 for prep., 2 for meas. pulses
+parse(p, varargin{:});
+type = p.Results.type;
+prep_meas = p.Results.prep_meas;
 
 switch nbrPulses
     case 4
         %Four pulse set
-        Uset1Q{1}=eye(2);
-        Uset1Q{2}=expm(-1i*(pi/4)*X);
-        Uset1Q{3}=expm(-1i*(pi/4)*Y);
-        Uset1Q{4}=expm(-1i*(pi/2)*X);
+        if strcmp(type, 'Clifford')
```

Comment:
@dieris  preference for switch/case of an if/else chain as it makes clear there is no priority